Vireo_VS/VireoCommandLine.vcxproj

```diff
@@ -170,4 +174,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Please add another line at the end to remove the "red character"

Reply:
Since these files are generated by IDE, I think it is likely ok to ignore the end of file character related diff.